recipes/pandemy/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "Pandemy" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: def2257d3044e3249ddde28b916db20ad4d9b751271beae5bea38a40c525d863
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
```

Comment:
The standard script command for Python packages is `script: {{ PYTHON }} -m pip install . -vv`.
`--no-deps` isn't needed as it is added automatically by conda-build.

No need to change. This is just a comment.

Reply:
Thanks for the feedback I will remove --no-deps in version 1.1.0.